README.md

```diff
@@ -10,6 +10,55 @@ Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs
 a single-node Kubernetes cluster inside a VM on your laptop for users looking
 to try out Kubernetes or develop with it day-to-day.
 
+## Requirements For Running Minikube
+* VirtualBox installation
+* VT-x/AMD-v virtualization must be enabled in BIOS
+
+## Usage
+
+Here's a brief overview of minikube usage:
+
+```shell
+$ minikube start
+Starting local Kubernetes cluster...
+Running pre-create checks...
+Creating machine...
+Starting local Kubernetes cluster...
+Kubernetes is available at https://192.168.99.100:443.
+
+$ $(minikube docker-env)
+$ docker build -t helloworld .
+Successfully built d16fe85e1abe
+$ kubectl run hello-minikube --image=helloworld --hostport=8000 --port=8080 --generator=run-pod/v1
+pod "hello-minikube" created
+$ curl http://$(minikube ip):8000
+Hello World!
+$ minikube stop
+Stopping local Kubernetes cluster...
+Stopping "minikubeVM"...
+```
+
+## Features
+ * Minikube packages and configures a Linux VM, Docker and all Kubernetes components, optimized for local development.
+ * Minikube supports Kubernetes features such as:
+   * DNS
+   * NodePorts
+   * ConfigMaps and Secrets
+   * Dashboards
+
+## Known Issues
+ * Features that require a Cloud Provider will not work in Minikube. These include:
+  * LoadBalancers
+  * PersistentVolumes
+  * Ingress
+ * Features that require multiple nodes. These include:
+  * Advanced scheduling policies
+  * DaemonSets
+ * Alternate runtimes, like rkt.
+
+If you need these features, don't worry! We're planning to add these to minikube over time. Please leave a note in the
+issue tracker about how you'd like to use minikube!
+
 ## Design
 
 Minikube uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) for provisioning VMs, and [localkube](https://github.com/redspread/localkube)
```

Comment:
While you're at it, we're not using Redspread's `localkube` anymore, maybe we should mention they donated it for continued development in this repo?